_guides/znc.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: documentation
+title: Set up ZNC with The Lounge
+---
+
+## What is ZNC?
+ZNC is an advanced IRC bouncer, which inevitably begs the question, what's a bouncer?
+
+Usually an IRC client connects directly to an IRC server. An IRC bouncer sits between the client (for example thelounge) and the irc server (for example Freenode).
+That allows it to add some additional functionality over a direct connection, some examples listed below
+
+- Attach multiple clients (say a computer, phone, tablet etc.) to the bouncer, all sharing the same nick.
+
+- Messages are saved even when no client is connected, playing back what was said when a client comes back
+
+- As the bouncer (preferably) never disconnects, mode changes, nick names and channels are kept at all times
+
+- Sharing block lists / using additional functionality across different clients, without having to configure each one of them
+
+## What thelounge gives you out of the box
+Thelounge already gives you most of the features you expect from a bouncer:
```

Comment:
The Lounge (brand name)

Reply:
also everywhere else it's used

Reply:
yeah, just wanted to mention that I'll fix it everywhere